### PR TITLE
added js implementation of debounce

### PR DESCRIPTION
**Pull request by rubiin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1643,7 +1644,30 @@ Create a new function that calls _func_ with _thisArg_ and _args_.
   7.0 ✔  |  ✔ | 4.0 ✔ |  9.0 ✔ |  11.6 ✔ |  5.1 ✔  |
 
  **[⬆ back to top](#quick-links)**
+ 
+ ### _.debounce
+Create a new function that calls _func_ with _thisArg_ and _args_.
+
+  ```js
+ const debounce = (callback, time = 250, interval) =>
```
Comment (ODudek) [inline README.md]: You should use function instead arrow function otherwise this won't work in IE

Comment (rubiin) [reply README.md]: sure

Comment (rubiin) [reply README.md]: @ODudek  done

File: README.md [hunk 2]
```diff
@@ -1643,7 +1644,37 @@ Create a new function that calls _func_ with _thisArg_ and _args_.
...
+		}, wait);
+		if (immediate && !timeout) func.apply(context, args);
+	};
+}
+
+// Avoid costly calculations while the window size is in flux.
+jQuery(window).on('resize', debounce(calculateLayout, 150));
+
+  ```
+#### Browser Support for `Function.prototype.bind()`
```
Comment (ODudek) [inline README.md]: Support for bind? Did you forget to update this line?

Comment (rubiin) [reply README.md]: @ODudek  fixed the typo

**Comment by ODudek** [comment]
For me code looks fine

**Comment by cht8687** [comment]
👍

**Comment by cjtaylor1990** [comment]
Looks good to me!

Sorry about my lack of activity on GitHub in the last week. I was incredibly busy with work and I had a big interview for a certain FAANG (that I may be actually getting an offer from).

**Comment by rubiin** [comment]
cool, no worries. I am also thinking of putting up a repo with snippet extensions for text editors like atom,vscode using the contents of the repo

**Comment by cht8687** [comment]
@rubiin This sounds like an awesome idea. 😄

**Pull request merged**